component/main.js

```diff
@@ -1,5 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import ClientCore from "./client/ClientCore.jsx";
+import * as firebase from "firebase";
+
+// Initialize Firebase
+var config = {
+  apiKey: "AIzaSyBakHeV8lMlysuBRtIWU9vz_hv6dF_zHxM",
+  authDomain: "url-connet.firebaseapp.com",
+  databaseURL: "https://url-connet.firebaseio.com",
+  projectId: "url-connet",
+  storageBucket: "url-connet.appspot.com",
+  messagingSenderId: "1089725560944"
+};
```

Comment:
Yeah, sorry. However, I don't think we can change the projectID after it is created. I updated the Project name and public-facing name.
https://console.firebase.google.com/project/url-connet/settings/general/
If you want to update this, then we would need to create a new firebase project.